### PR TITLE
Enhancement - Clearing boilerplate

### PR DESCRIPTION
**Pull request by elroytoscano**
Switched to yarn from npm 
Updated App.js - cleared out boilerplate code
Updated readme
Added more info to readme

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,24 +2,7 @@ import logo from './logo.svg';
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
```
Comment (Raman-Iyer) [inline src/App.js:21]: Don't remove this code as it is the boiler plate code

File: src/App.js [hunk 2]
```diff
@@ -2,24 +2,7 @@ import logo from './logo.svg';
...
-  );
+  return <>Flow_2</>;
```
Comment (Raman-Iyer) [inline src/App.js]: Add this in a tag instead of blank tags

**Review by Raman-Iyer — CHANGES_REQUESTED** [review]
Check the comments and make the necessary changes

File: src/App.js (hunk 2 above)
Comment (elroytoscano) [reply src/App.js]: Added tags

File: src/App.js (hunk 1 above)
Comment (elroytoscano) [reply src/App.js:21]: Since I'm working on creating a template for CRA, removed the boilerplate code.

**Pull request merged**